simulation_parameters/microbe_stage/bio_processes.json

```diff
@@ -39,7 +39,7 @@
     },
     "Outputs": {
       "oxygen": 3,
-      "glucose": 0.04
+      "glucose": 0.03
```

Comment:
I'm kind of questioning this as most of the player feedback about photosynthesis recently has been that it is totally useless. So nerfing it is a bit questionable, though it seems basically al of that feedback *has* been with experimental mode enabled.

Reply:
I'm going to further playtesting with photosynthetic life to make a judgement. The logic here though was that since glucose burn was slowed down, photosynthesis could be buffed, so limiting glucose production a bit could help counter that.